prawcore/rate_limit.py

```diff
@@ -38,10 +41,12 @@ def delay(self):
         """Sleep for an amount of time to remain under the rate limit."""
         if self.next_request_timestamp is None:
             return
-        sleep_seconds = self.next_request_timestamp - time.time()
-        if sleep_seconds <= 0:
+        stime = self.next_request_timestamp - time.time()
```

Comment:
Can you keep the variable names as `sleep_seconds`?

Reply:
yup - can retain the variable names. Do you want them both unified to ``sleep_seconds``?

Reply:
Yeah that seems to make more sense than the other `sleep_time`. Good catch.